app/components/post_profile_picture/index.js

```diff
@@ -19,12 +20,14 @@ function mapStateToProps(state, ownProps) {
     const post = ownProps.post;
     const user = getUser(state, post.user_id);
 
+    const overrideIconURL = Client4.getAbsoluteUrl(post?.props?.override_icon_url); // eslint-disable-line camelcase
```

Comment:
Out of curiosity, is the reason this doesn't "just work" because of the execution context of RN? (I'm more familiar with React apps where a relative URL would, of course, just load normally without more configuration.)

Reply:
@lieut-data relative URL's just does not work, the device needs the absolute URL as the is nothing caching the url address for you, that is why on the `Client4` object we store the server url as well as in an entity in the redux store.